gatsby/create-node/create-preview-cards.ts

```diff
@@ -46,16 +45,12 @@ const createPreviewCard = async (
   }
 };
 
-const createPreviewCards = async ({ node, ...rest }) => {
+export const createPreviewCards = async ({ node, ...rest }) => {
   if (node.internal.type === 'SyPersonioJob') {
-    await createPreviewCard(node.name, { node, ...rest });
+    await createPreviewCard(node.name, { node, ...rest } as any);
```

Comment:
Could this be easily be resolve by adding proper types to `export const createPreviewCards = async ({ node, ...rest })`? Or too much effort?

Reply:
Yep, got it working 👍